server/src/test/java/io/crate/integrationtests/CopyIntegrationTest.java

```diff
@@ -914,4 +915,95 @@ public void test_copy_from_unknown_column_to_dynamic_object() throws Exception {
         execute("select o['a'] + o['b'] from t");
         assertThat(printedTable(response.rows()), is("579\n"));
     }
+
+    @Test
+    public void testCopyFromWithValidationSetToFalseIgnoresTypeValidation() throws Exception {
+
+        // copying an empty string to a boolean column
+
+        execute("create table t (a boolean)");
+
+        List<String> lines = List.of("{\"a\": \"\"}");
+        File file = folder.newFile(UUID.randomUUID().toString());
+        Files.write(file.toPath(), lines, StandardCharsets.UTF_8);
+
+        execute("copy t from ? with (shared = true, validation = false) return summary",
+                new Object[]{Paths.get(file.toURI()).toUri().toString()});
+        execute("refresh table t");
+        execute("select a from t");
+        assertThat(response.rows()[0][0], is(false)); // empty string inserted as a false, implying validation skipped
+    }
```

Comment:
If the original behaviour is recovered, this issue will also come back("" inserted as false). But it could be just a limitation of not validating src data.